### PR TITLE
Backport BHSawesome edits.

### PR DESCRIPTION
**Pull request by gigamonkey**

### CODE REVIEW

File: _sources/Unit1-Getting-Started/topic-1-3-variables.rst [hunk 1]
```diff
@@ -54,15 +54,29 @@ Data Types
...
-    -  **int** - which stores integers (numbers like 3, -76, 0, 20393 with no decimal or fractional parts).
-
-    -  **double** - which stores double precision floating point numbers (decimal numbers like 6.3, -0.9, and 60293.93032). Floating point refers to the decimal point which "floats" or moves around in the decimal number. The type ``double`` is named after the fact that it holds double the number of bits as the type ``float`` (which is not used as often). 
-
-    -  **boolean** - which stores boolean values (either `true` or `false` values). The type `boolean` is named after `George Bool <https://en.wikipedia.org/wiki/George_Boole>`_, the inventor of Boolean algebra. 
-
-**String** is one of the object types on the exam and is the name of a class in Java.  A *string* object has a sequence of characters enclosed in a pair of double quotes - like "Hello".  You will learn more about ``String`` objects in Unit 2.
-
-A type is a set of values (a domain) and a set of operations on them. For example, you can do addition operations with ints and doubles but not with booleans and Strings.
+- ``int`` which represent integers, i.e. whole numbers like 3, -76, 20393.
```
Comment (bhoffman0) [inline _sources/Unit1-Getting-Started/topic-1-3-variables.rst:57]:
```suggestion
- ``int`` which represent integers, i.e. numbers like 3, 0, -76, 20393.
```

Comment (bhoffman0) [reply _sources/Unit1-Getting-Started/topic-1-3-variables.rst:57]: A math teacher told me integers and whole numbers are not the same.

Comment (bhoffman0) [reply _sources/Unit1-Getting-Started/topic-1-3-variables.rst:57]: Also should it be represents with an s?

**Review by bhoffman0 — APPROVED** [review]

File: _sources/Unit1-Getting-Started/topic-1-3-variables.rst (hunk 1 above)
Comment (gigamonkey) [reply _sources/Unit1-Getting-Started/topic-1-3-variables.rst:57]: I guess that's right about the whole numbers. Looking it up natural numbers are 1, 2, 3, ..., whole numbers are the natural numbers and 0, and integers are the whole numbers plus the negatives of the natural numbers.

**Pull request merged**